templates/pages/movie.html.twig

```diff
@@ -0,0 +1,161 @@
+                        @click.outside="rateMenuOpen = false"
+                    >
+                        <p class="text-gray-300">{{ movie.title }}</p>
+                        {% include "components/rate-menu.html.twig" %}
+                        <div class="mt-5 sm:mt-6">
+                            <button type="button" class="inline-flex w-full justify-center rounded-md bg-indigo-600 px-6 py-3 text-lg font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
+                                Rate
+                            </button>
+                        </div>
+                    </div>
+                </div>  
+            </div>
+        </div>
+
+        {# Review menu #}
+        <div class="relative z-10 text-white" aria-labelledby="modal-title" role="dialog" aria-modal="true" x-show="reviewMenuOpen">
+            <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
+            <div class="fixed inset-0 z-10 w-screen overflow-y-auto">
+                <div class="flex min-h-full items-end justify-center p-4 text-center sm:items-center sm:p-0">
+                    <div class="relative transform overflow-hidden rounded-lg bg-gray-900 p-4 text-left shadow-xl transition-all sm:my-8 w-full sm:max-w-xl md:max-w-2xl sm:p-6"
+                        @click.outside="reviewMenuOpen = false"
+                    >
+                        <form @submit.prevent class="flex flex-col gap-6">
+                            <div>
+                                <label for="title" class="block text-sm font-medium leading-6">Title</label>
+                                <div class="mt-2">
+                                    <input type="text" name="title" id="title" class="bg-gray-800 block w-full rounded-md border-0 py-1.5 text-gray-300 shadow-sm ring-1 ring-inset  placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                                </div>
+                            </div>
+                        
+                            <div>
+                                <label for="comment" class="block text-sm font-medium leading-6">Review</label>
+                                <div class="mt-2">
+                                    <textarea rows="4" name="comment" id="comment" class="bg-gray-800 block w-full rounded-md border-0 py-1.5 text-gray-300 shadow-sm ring-1 ring-inset  placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"></textarea>
+                                </div>
+                            </div>
+
+                            <div name="rating" class="mt-2">
+                                {% include "components/rate-menu.html.twig" %}
```

Comment:
Here menu as well?

Reply:
I thought having the option to give a rating when a submitting a review would be suitable.

Reply:
I am talking more about the naming of the component. These rating stars don't seem like a menu. The wording is confusing